nautobot/docs/configuration/authentication/sso.md

```diff
@@ -36,6 +36,10 @@ $ pip3 install "social-auth-core[openidconnect]"
 
 #### SAML
 
+!!! note
+    `social-auth-core-saml` has system dependencies that must be installed *before* the python packages.
+    Please follow the [installation instructions](https://github.com/mehcode/python-xmlsec) for your platform.
+
```

Comment:
I'm actually working on expanding the SAML docs. Please go ahead and remove this change.